aerobridge/settings.py

```diff
@@ -164,6 +166,6 @@
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
-  
+FIXTURE_DIRS = [os.getcwd() + '/tests/fixtures/']
```

Comment:
Recommend adding a comment to tell what these will do, for new members to the project

Reply:
I tried that. Actually this variable is used to populate database with dummy data and the paths are absolute system paths instead of django app's relative path. That's why had to get `cwd`.